introduction.md

```diff
@@ -7,5 +7,8 @@ Em, JavaScript.
 ````
 printf!('Hello, world!')
 ````
+* Git
+* JavaScript
+* Ruby
 
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?